src/vumi2/config.py

```diff
@@ -146,10 +146,15 @@ def load_config_from_file(path: Path) -> dict[str, Any]:
     Loads a config from a yaml file, if it exists, otherwise returns an empty
     dictionary.
     """
-    if path.exists():
-        with path.open() as f:
-            return yaml.safe_load(f)
-    return {}
+    # FIXME: Ideally we'd only ignore missing configs if the VUMI_CONFIG_FILE
```

Comment:
Could we just force this and do a major release?

Reply:
We could, and I added that comment partly to start the conversation around that. It's not a decision I want to make unilaterally, though.

Reply:
Cool, I think it makes sense.

Reply:
On further reflection, pretty much everything about the default config file path was problematic. Since this was probably a breaking change anyway, I went ahead and removed the default entirely with a version bump to `0.3.0.dev0`.